__tests__/data/google_adwords_enhanced_conversions_output.json

```diff
@@ -195,9 +200,10 @@
     "statusCode": 400,
     "error": "[Google_adwords_enhanced_conversions]:: Conversion named Product Viewed is not exist in rudderstack dashboard",
     "statTags": {
-      "destination": "google_adwords_enhanced_conversions",
-      "stage": "transform",
-      "scope": "exception"
+      "category": "",
```

Comment:
empty tag required?

Reply:
For consistency in queries, added an empty string(ed) category